### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by martinmona**
## Summary
- Completed Navbar Animation

## Screenshot
Peek 2019-08-01 14-28

## Trello Card
https://trello.com/c/EuGZBuMe/36-animations

### CODE REVIEW

File: Maquetado/scss/navbar.scss [hunk 1]
```diff
@@ -12,12 +24,14 @@
...
   margin: 0 auto;
-  max-height: 100%;
-  position: sticky;
 }
 
 .nav-logo {
+  animation-duration: 3s;
+  animation-name: slidein;
   height: 70px;
+  max-height: 100%;
```
Comment (manueltuero) [inline Maquetado/scss/navbar.scss]: You can delete this line

File: Maquetado/scss/navbar.scss [hunk 2]
```diff
@@ -12,12 +24,14 @@
...
+  position: sticky;
```
Comment (manueltuero) [inline Maquetado/scss/navbar.scss]: Same

File: Maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -12,12 +24,14 @@
   @include shadow;
   justify-content: space-evenly;
   margin: 0 auto;
-  max-height: 100%;
-  position: sticky;
 }
 
 .nav-logo {
+  animation-duration: 3s;
```
Comment (manueltuero) [inline Maquetado/scss/navbar.scss]: You can use a shorthand here

File: Maquetado/scss/details.scss [hunk 4]
```diff
@@ -140,49 +127,56 @@
...
+  grid-template-areas: 'a b c d''e f g h';
   justify-content: center;
   margin: 20px auto;
 }
 
-.list-book-a {
-  @extend %list-book;
-  grid-area: a;
-}
+.list-item {
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Maybe you can make this using a sass `for` and lists. If you want.

File: Maquetado/scss/navbar.scss [hunk 5]
```diff
@@ -27,11 +27,8 @@
 }
 
 .nav-logo {
-  animation-duration: 3s;
-  animation-name: slidein;
+  animation: slidein 3s;
```
Comment (manueltuero) [inline Maquetado/scss/navbar.scss]: :+1:

File: Maquetado/scss/details.scss (hunk 4 above)
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Maybe you can make this using a sass `for` and lists. If you want.

File: Maquetado/scss/navbar.scss [hunk 6]
```diff
@@ -2,6 +2,18 @@
   border-top: 7px solid $color;
 }
 
+@keyframes slidein {
+  from {
+    margin-left: 100%;
```
Comment (henryzarza) [inline Maquetado/scss/navbar.scss]: Isn't a good practice anime this properties. Check this [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108)

File: Maquetado/scss/navbar.scss [hunk 7]
```diff
@@ -2,6 +2,18 @@
...
+  }
+  75% {
+    margin-left: -100px;
```
Comment (henryzarza) [inline Maquetado/scss/navbar.scss]: ☝️

File: Maquetado/scss/navbar.scss [hunk 8]
```diff
@@ -2,6 +2,18 @@
...
+  }
+  to {
+    margin-left: 0;
```
Comment (henryzarza) [inline Maquetado/scss/navbar.scss]: ☝️

File: Maquetado/scss/navbar.scss [hunk 9]
```diff
@@ -2,6 +2,18 @@
...
 
+@keyframes slidein {
+  from {
+    transform: translateX(100%);
+  }
+  75% {
+    transform: translateX(-100%);
+  }
+  to {
+    transform: none;
```
Comment (henryzarza) [inline Maquetado/scss/navbar.scss]:
```suggestion
    transform: translateX(0);
```

**Pull request merged**